packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -0,0 +1,34 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import type { ProgressSlots, ProgressState } from './Progress.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+
+export const progressClassName = 'fui-Progress';
```

Comment:
You should remove this since we are now using the `<component>ClassNames` object.